public/app/features/alerting/components/PreviewQueryTab.tsx

```diff
@@ -24,36 +26,45 @@ export const PreviewQueryTab: FC<Props> = ({ data, height, width }) => {
   // Select padding
   const padding = 16;
 
-  if (!data?.series?.length) {
-    return <Table data={toDataFrame([])} height={height} width={width} />;
+  if (!data) {
+    return (
+      <EmptyState title="Run queries to view data.">
+        <Button onClick={onRunQueries}>Run queries</Button>
+      </EmptyState>
+    );
   }
 
-  if (data.series.length > 1) {
-    return (
-      <div className={styles.wrapper}>
-        <div style={{ height: height - theme.spacing.formInputHeight - 16 }}>
-          <Table
-            data={data.series[currentSeries]}
-            height={height - theme.spacing.formInputHeight - padding}
-            width={width}
-          />
+  if (data.series) {
+    if (data.series.length > 1) {
+      return (
+        <div className={styles.wrapper}>
+          <div style={{ height: height - theme.spacing.formInputHeight - 16 }}>
+            <Table
+              data={data.series[currentSeries]}
+              height={height - theme.spacing.formInputHeight - padding}
+              width={width}
+            />
+          </div>
+          <div className={styles.selectWrapper}>
+            <Select
+              onChange={(selectedValue) => setSeries(selectedValue.value!)}
+              options={series}
+              value={currentSeries}
+            />
+          </div>
         </div>
-        <div className={styles.selectWrapper}>
-          <Select
-            onChange={(selectedValue) => setSeries(selectedValue.value!)}
-            options={series}
-            value={currentSeries}
-          />
-        </div>
-      </div>
-    );
+      );
+    } else if (data.series.length > 0) {
```

Comment:
This if statements seem weird to me the first is greater than 1 and the second is greater than 0? Can we simplify this?